test_common/harness/imageHelpers.cpp

```diff
@@ -815,17 +815,17 @@ void get_max_sizes(
         {
 
             // Determine the size of the fixed dimension
-            size_t M = maximum_sizes[fixed_dim];
-            size_t A = max_pixels;
+            double M = static_cast<double>(maximum_sizes[fixed_dim]);
```

Comment:
This causes a change in behavior of the `A / M` divide below, is that intended? If so, it should probably be called out in its own PR. 
To preserve the integer divide, we could just get rid of the double types altogether by replacing `fmin` with `min`.